GDJS/Runtime/events-tools/networktools.js

```diff
@@ -141,7 +141,9 @@ gdjs.evtTools.network.objectVariableStructureToJSON = function (
 };
 
 gdjs.evtTools.network._objectToVariable = function (obj, variable) {
-  if (!isNaN(obj)) {
+  if (obj == null) {
```

Comment:
Always use `===`, because this will also match undefined. Use `=== undefined` in a separate if branch if you need to.